hrs-portlets-webapp/src/main/java/edu/wisc/portlet/hrs/web/HrsControllerBase.java

```diff
@@ -64,6 +73,24 @@ public final String getNavLinks(PortletRequest request) {
     }
     
     /**
+     * The generic Message if you want to override the default in 
+     * messages.properties
+     */
+    @ModelAttribute("genericErrorMessage")
+    public final String getGenericErrorMessage(PortletRequest request){
```

Comment:
I think as implemented this is going to read `messages.properties` from disk on every request?  Performance worth optimizing?  At the least maybe only read the `messages.properties` default if the preference isn't set?